app/models/volunteer_op.rb

```diff
@@ -34,4 +42,16 @@ def self.contains_description?(text)
   def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
+  
+  def address_complete?
+    address.present? && postcode.present?
+  end 
+  
+  private
+  
+  def clear_lat_lng
+    return if address_complete?
+    self.longitude = nil
+    self.latitude = nil
+  end
```

Comment:
@tansaku it's better to persist doit volunteer op coordinates directly without requiring geocoder to regenerate them.
I agree that using an after_save hook seems to be a cleaner solution. Maybe we should plan it as enhancement ?